<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
   </widget>

   <!-- BUTTONS -->
   <widget class="QPushButton" name="btn7"><property name="text"><string>7</string></property><property name="geometry"><rect><x>20</x><y>80</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btn8"><property name="text"><string>8</string></property><property name="geometry"><rect><x>80</x><y>80</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btn9"><property name="text"><string>9</string></property><property name="geometry"><rect><x>140</x><y>80</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btnDiv"><property name="text"><string>/</string></property><property name="geometry"><rect><x>200</x><y>80</y><width>50</width><height>40</height></rect></property></widget>

   <widget class="QPushButton" name="btn4"><property name="text"><string>4</string></property><property name="geometry"><rect><x>20</x><y>130</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btn5"><property name="text"><string>5</string></property><property name="geometry"><rect><x>80</x><y>130</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btn6"><property name="text"><string>6</string></property><property name="geometry"><rect><x>140</x><y>130</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btnMul"><property name="text"><string>*</string></property><property name="geometry"><rect><x>200</x><y>130</y><width>50</width><height>40</height></rect></property></widget>

   <widget class="QPushButton" name="btn1"><property name="text"><string>1</string></property><property name="geometry"><rect><x>20</x><y>180</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btn2"><property name="text"><string>2</string></property><property name="geometry"><rect><x>80</x><y>180</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btn3"><property name="text"><string>3</string></property><property name="geometry"><rect><x>140</x><y>180</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btnSub"><property name="text"><string>-</string></property><property name="geometry"><rect><x>200</x><y>180</y><width>50</width><height>40</height></rect></property></widget>

   <widget class="QPushButton" name="btn0"><property name="text"><string>0</string></property><property name="geometry"><rect><x>20</x><y>230</y><width>110</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btnMod"><property name="text"><string>%</string></property><property name="geometry"><rect><x>140</x><y>230</y><width>50</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btnAdd"><property name="text"><string>+</string></property><property name="geometry"><rect><x>200</x><y>230</y><width>50</width><height>40</height></rect></property></widget>

   <widget class="QPushButton" name="btnClear"><property name="text"><string>C</string></property><property name="geometry"><rect><x>20</x><y>280</y><width>110</width><height>40</height></rect></property></widget>
   <widget class="QPushButton" name="btnEqual"><property name="text"><string>=</string></property><property name="geometry"><rect><x>140</x><y>280</y><width>110</width><height>40</height></rect></property></widget>

  </widget>
 </widget>
</ui>
